pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -401,13 +403,15 @@ public static void main(String[] args) throws Exception {
 
             long now = System.nanoTime();
             double elapsed = (now - oldTime) / 1e9;
-            double rate = messagesReceived.sumThenReset() / elapsed;
+            long total = messagesReceived.sumThenReset();
```

Comment:
is `total` all messages that have been consumed? just wondering if `.sumThenReset` will actually do what it would expected to do, reset to 0 after returning the value?

Reply:
@lhotari this is really a good point.
I tested it and it looks like it work very well, I mean the number is never reset, it is a always increasing value

Reply:
Thanks for the clarification.

Reply:
@lhotari I have retested it and you were right.

I updated the patch.

good catch !